doc/backup.md

```diff
@@ -5,6 +5,8 @@ To backup a LXD instance different strategies are available.
 ## Full backup
 This requires that the whole `/var/lib/lxd` folder will be backuped up.
 Additionally, it is necessary to backup all storage pools as well.
+The filesystem used for backups must support symlinks.
```

Comment:
Symlinks are usually not the worst thing to restore after you accidentally copy to fat, file permissions, xattrs, ... are a lot harder to figure out and much more annoying to restore :)

Anyway, still feel that we should drop that line as it's really common sense and at the same time, trying to list all the fs features one might need is going to be pretty much impossible as it depends on what they're using in the container.